michaeljones/breathe
# Update flake8 and mypy stuff

- Remove ``#noqa`` comments
- Switch comment-based type annotation to variable type hints
- Make mypy warn about redundant things
- General mypy fixes

There is still one error left:
```
breathe/renderer/sphinxrenderer.py:439: error: List item 0 has incompatible type "Tuple[str]"; expected "Tuple[str, str]"
```
